O365DataRetriever.ps1

```diff
@@ -630,9 +631,11 @@ $ConnectButton.Add_Click( {
 		#Check ExecutionPolicy is set to "Unrestricted" on computer running the tool (for version 0.1.0)
 		$ExecutionPolicy = Get-ExecutionPolicy
 		if($ExecutionPolicy -ne "Unrestricted"){
-			Write-Host "For this release 0.1.0, the execution policy must be set to Unrestricted. Please change it and try again." -ForegroundColor White -BackgroundColor DarkRed
-			$Window.Close()
-			break
+            Write-Host "For this release 0.1.0, the execution policy must be set to Unrestricted. Current Setting $ExecutionPolicy." -ForegroundColor White -BackgroundColor DarkRed
+            Write-Host "We will set it to Unrestricted for this process but not more!" -ForegroundColor Green
```

Comment:
I don't like the idea of the tool setting the Execution Policy to "unrestricted". I prefer people to set it themselves after reading the pre-reqs and acknowledging it. Or not running the tool if there's security concerns.